userspace/libscap/scap_suppress.h

```diff
@@ -33,6 +33,13 @@ typedef struct scap_tid
 	UT_hash_handle hh; ///< makes this structure hashable
 } scap_tid;
 
+// Famous last words: will never have more than 1024 CPUs... :-)
+#define SCAP_CPUID_MAX (1024)
```

Comment:
This is at the same time:
* massive overkill (for a typical system)
* not nearly enough (the largest machines have had more cpus for years now)

Reply:
> massive overkill (for a typical system)

Yes, it would be more elegant to allocate the memory dynamically, but...
This would mean doing an extra cache line fetch (for the dynamically allocated memory) for each lookup at run-time = bad performance :-(

> not nearly enough (the largest machines have had more cpus for years now)

Good catch! I pushed a change tolerating more than 1024 CPUs :-)